python/context.cpp

```diff
@@ -136,8 +124,10 @@ void register_contexts(pybind11::module& m) {
              "bind_procs"_a=false,
              "bind_threads"_a=false,
              "Construct an allocation with arguments:\n"
-             "  threads: The number of threads available locally for execution. Must be set to 1 at minimum. 1 by default.\n"
-             "  gpu_id:  The identifier of the GPU to use, None by default.\n")
+             "  threads:      The number of threads available locally for execution. Must be set to 1 at minimum. 1 by default.\n"
```

Comment:
Might this be a symbolic constant or string? Such as 'avail_threads'?

Reply:
Available as `arbor.env.default_concurrency()`.